dataservice/api/biospecimen/models.py

```diff
@@ -79,6 +79,10 @@ class Biospecimen(db.Model, Base):
                                           'Uber-anatomy ontology which '
                                           'represents harmonized anatomical'
                                           ' ontologies')
+    duo_id_biospecimen = db.Column(db.Text(),
```

Comment:
Why not just `duo_id` since it's already on the biospecimen?

Reply:
🤷‍♀ I just took it from the ticket: https://github.com/kids-first/kf-api-dataservice/issues/541#issue-504742357

Yea I guess `_biospecimen` is redundant though